core/templates/components/state-editor/state-editor.directive.ts

```diff
@@ -130,6 +140,16 @@ angular.module('oppia').directive('stateEditor', [
             $scope.interactionIdIsSet = false;
             $scope.servicesInitialized = false;
             $scope.stateName = StateEditorService.getActiveStateName();
+            ctrl.explorationId = ContextService.getExplorationId();
+            ctrl.explorationUrl = UrlInterpolationService.interpolateUrl(
+              '/createhandler/data/<exploration_id>', {
+                exploration_id: ctrl.explorationId
+              });
+            $http.get(ctrl.explorationUrl)
+              .then(function(response) {
```

Comment:
Please add type for this reponse.

Reply:
Shouldn't HTTP requests be made from backend API services? +cc @DubeySandeep. Are lint checks not working for this?

Reply:
I don't see a good reason not to... these are compile-time constructs only. Plus doing it now itself saves some time when we are migrating this file. Not doing this now adds an extra layer of us investigating or asking the contributor who introduced this. It's just better to ensure that we add types to any part of the code we modify (if we know what type that variable is).

Reply:
Done. PTAL.